src/services/usersService.js

```diff
@@ -14,9 +22,41 @@ const signUp = async (email, password, points) => {
 
   const hashPassword = await bcrypt.hash(password, salt);
 
-  return await usersDao.signUp(email, hashPassword, points);
+  return await usersDao.signUp(
+    email,
+    hashPassword,
+    firstName,
+    lastName,
+    subscription,
+    points
+  );
+};
+
+const logIn = async (email, password) => {
+  await pwValidationCheck(password);
+  await emailValidationCheck(email);
+
+  const [info] = await usersDao.logIn(email);
```

Comment:
변수명은 어떤 데이터를 나타내는지를 표현하는 것이 좋습니다.
```suggestion
  const user = await usersDao.logIn(email);
```